release-notes.md

```diff
@@ -8,12 +8,26 @@ Here are the release notes for the Experience Platform Mobile SDKs and Adobe Pla
 
 The following updates were made in this release:
 
+**iOS Core 2.3.5**
+
+* Added support for attach data rules consequence.
+* Added support for a boolean-type comparison for `Exist` or `Not Exist` rules condition.
```

Comment:
Please add  a "the" between "for" and "`Exist`" so that it reads:
"...comparison for the `Exist` or..."

(this change also applies to the similar bullet below)